steps/compliance/have-a-mrn/HaveAMRN.js

```diff
@@ -24,16 +24,6 @@ class HaveAMRN extends Question {
         );
     }
 
-    answers() {
-
-        return [];
-    }
-
-    values() {
-
-        return {};
-    }
-
```

Comment:
You shouldn't be removing these functions. The answers() function impacts what you see on CYA, the values() function impacts the JSON that's used for the API call. By removing these functions you are adding unnecessary noise to both CYA and the JSON.
**CYA**
<img width="752" alt="not-required-in-cya" src="https://user-images.githubusercontent.com/964268/34249230-f2336a9c-e630-11e7-8fd2-0c8a4e78e38d.png">
**JSON**
<img width="398" alt="not-required-in-json" src="https://user-images.githubusercontent.com/964268/34249255-034d949c-e631-11e7-892d-d94cd48dbdd7.png">

Reply:
fixed.